### ISSUE TITLE
MJPEG Stream Viewer URLs does not save & (ampersand) properly

### ISSUE THREAD

**Issue by tylertian123** [wpilibsuite/SmartDashboard#119]
The XML generated from an MJPEG stream viewer does not handle ampersands in its URL properly. 

To Reproduce: 
* Add an MJPEG Stream Viewer
* In the URL, enter something with an ampersand, e.g. "http://10.61.35.19:1180/stream?topic=/main_camera/image_raw&quality=60&width=960&height=540"
* Save the layout

This layout does not load properly. Upon further inspection, the property is saved literally, like this:
```xml
<property name="MJPG Server URL" value="http://10.61.35.19:1180/stream?topic=/main_camera/image_raw&quality=60&width=960&height=540"/>
```
However, as this is an XML document, ampersands start an escape sequence, thus the snippet shown above is not valid XML. To solve this problem, the "&" character (as well as other illegal XML characters such as "<") must be replaced with its corresponding escape sequence ("&amp;").

**Comment by tylertian123** [comment]
https://www.ibm.com/support/knowledgecenter/en/SSS28S_8.1.0/XFDL/i_xfdl_g_computes_illegal_characters_in_xml_attributes.html

**Comment by PeterJohnson** [comment]
Fixed in #124.
